python/employee-scheduling/src/employee_scheduling/demo_data.py

```diff
@@ -51,7 +50,7 @@ def generate_demo_data() -> EmployeeSchedule:
     initial_roster_length_in_days = 14
     start_date = earliest_monday_on_or_after(date.today())
 
-    random = Random(0)
+    random = Random(37)
```

Comment:
Yes, I admit. This is a dirty hack because random in Java and Python behave differently, and it turns out that seed 0 gives a feasible problem in Java, but an infeasible in Python. While 37 gives a feasible one in Python.